captum/attr/_core/layer/layer_gradient_shap.py

```diff
@@ -96,40 +96,29 @@ def attribute(
                         that for all given input tensors, dimension 0 corresponds
                         to the number of examples, and if multiple input tensors
                         are provided, the examples must be aligned appropriately.
-            baselines (scalar, tensor, tuple of scalars or tensors, callable, optional):
+            baselines (tensor, tuple of tensors, callable):
                         Baselines define the starting point from which expectation
-                        is computed and can be provided as:
+                        is computed can be provided as:
```

Comment:
nit: seems like leaving and is grammatically correct here? (also in GradientShap and NeuronGradientShap as well?). Or otherwise can make a separate sentence?